### ISSUE TITLE
Balancer

### ISSUE THREAD

**Issue by mauriliond** [jc21/nginx-proxy-manager#309]
An application is the best that has been tested to manage NGINX, only part of the configuration of the balancers is missing.

**Comment by jc21** [comment]
Yep definitely would be a nice enhancement, just need to come up with a UI that meets the goal of being easy for noobs and powerful for experts.

**Comment by chaptergy** [comment]
Closing this in favor of https://github.com/jc21/nginx-proxy-manager/issues/156
